behaviours/meme_rater.py

```diff
@@ -311,23 +311,46 @@ async def delete_meme(event: hikari.GuildReactionAddEvent) -> None:
         net_shit_count += 1
 
     if net_shit_count >= int(os.getenv("MEME_VOTE_DELETE_THRESHOLD", 4)):
-        response = await event.app.rest.create_message(
+        await delete_meme(event, message, age, shit_reaction, ten_reaction)
+
+    raise behaviours.EndProcessing()
+
+
+async def delete_meme(
+    event: hikari.GuildReactionEvent,
+    message: hikari.Message,
+    age: timedelta,
+    shit_reaction: hikari.Reaction,
+    ten_reaction: hikari.Reaction | None,
+):
+    if MEME_DELETE_LOG_CHANNEL_ID:
+        dislikers = await voter_names(event, message, shit_reaction)
```

Comment:
`shitters` and `non_shitters`

Reply:
`shitters` is ambiguous, it could either be the ones deeming something shit, or the ones producing shit.

Reply:
`shit_reacters` then, pedant.

Reply:
that feels a bit too verbose